content/en/blog/_posts/2025-04-23-jobs-successpolicy-goes-ga.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: blog
+title: "Kubernetes 1.33: Job's SuccessPolicy Goes GA"
+date: 2025-04-23
+slug: kubernetes-1-33-jobs-success-policy-goes-ga
+authors: >
+  [Yuki Iwai](https://github.com/tenzen-y) (CyberAgent, Inc)
+---
+
+We're pleased to announce that Job's __Success Policy__ will  graduate to General Availability (GA) as part of Kubernetes v1.33 release.
+
+## About Job's Success Policy
+
+In batch workloads, you might want to use leader-follower patterns like [MPI](https://en.wikipedia.org/wiki/Message_Passing_Interface),
+in which the leader controls the execution, including the followers' lifecycle.
+
+In this case, you might want to mark it as succeeded
+even if some the indexes failed. Unfortunately, the Kubernetes Job without __Success Policy__, in most cases,
+requires all Pods to finish successfully for a Job to reach succeeded state.
+
+Job's __Success Policy__ allows you to specify the early exit criteria using the `.spec.successPolicy` 
+field only for the [Indexed Job](/docs/concept/workloads/controllers/job/#completion-mode). 
+Which describes a set of rules either using a list of succeeded indexes for a job, or defining a minimal required size of succeeded indexes. 
+
+This feature is especially useful for the scientific simulation workloads, AI/ML and High-Performance computing batch workloads. 
+In those workloads users perform various simulations and might not necessarily care about all the experiments to complete successfully,
+rather they require only a certain number of them to finish successfully. 
+In this case, the leader index failure is only the Job's failure criteria, and workers results is not relevant.
```

Comment:
```suggestion
In this case, the leader index failure is only the Job's failure criteria, and workers results are relevant
only indirectly via the status of the leader index.
```
I think saying "workers results is not relevant." may be a bit misleading. They are relevant, but indirectly. wdyt?

Reply:
That makes sense. Actually, I imagined the followers perform only sshd daemon. So, I said, `It is not relevant. '
However, the failure of sshd launching might indirectory relevant the JOB failure.